src/app/[locale]/(default)/rules/[subset]/page.tsx

```diff
@@ -0,0 +1,15 @@
+import { rulesMockdata } from '@/mock-data/rules';
+
+export default function SubPage({
+  params: { subset },
+}: { params: { subset: string } }) {
```

Comment:
every single page and layout needs the unstable_setRequestLocale, including this one. It allows for static generation of pages